flamingo_tools/s3_utils.py

```diff
@@ -0,0 +1,107 @@
+import os
+
+import s3fs
+import zarr
+
+from tqdm import tqdm
+
+# Using incucyte s3 as a temporary measure.
+MOBIE_FOLDER = "/mnt/lustre-emmy-hdd/projects/nim00007/data/moser/lightsheet/mobie"
+SERVICE_ENDPOINT = "https://s3.gwdg.de/"
+BUCKET_NAME = "incucyte-general/lightsheet"
+
+# For MoBIE:
+# https://s3.gwdg.de/incucyte-general/lightsheet
+
+def check_s3_credentials(bucket_name, service_endpoint, credentials):
+    """
+    Check if S3 parameter and credentials were set either as a function input or were exported as environment variables.
+    """
+    if bucket_name is None:
```

Comment:
Shouldn't we use the global variables defined in the beginning of the file as the default fall back here? Otherwise what's their purpose?